sdcm/cluster_cloud.py

```
@@ -353,25 +357,88 @@ def db_up(self):
+            )
+            return False
+
+        return True
+
+    def _self_install_vector(self):
+        """Self-install vector.dev from a locally-downloaded .deb."""
+        arch_result = self.remoter.run("dpkg --print-architecture", retry=0)
+        arch = arch_result.stdout.strip() if arch_result.return_code == 0 else "amd64"
+
+        local_package = download_vector_locally(arch=arch)
+        remote_path = f"/tmp/{os.path.basename(local_package)}"
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)